dspace-api/src/main/java/org/dspace/authority/orcid/Orcidv2.java

```diff
@@ -0,0 +1,145 @@
+package org.dspace.authority.orcid;
+
+import org.apache.commons.lang.StringUtils;
+import org.apache.http.HttpResponse;
+import org.apache.http.client.HttpClient;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.impl.client.HttpClientBuilder;
+import org.dspace.authority.AuthorityValue;
+import org.dspace.authority.SolrAuthorityInterface;
+import org.dspace.authority.orcid.xml.XMLtoBio;
+import org.dspace.authority.orcid.xml.XMLtoWork;
+import org.dspace.authority.rest.RESTConnector;
+import org.json.JSONObject;
+import org.orcid.jaxb.model.record_v2.Person;
+import org.orcid.jaxb.model.record_v2.Work;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.net.URLEncoder;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by jonas - jonas@atmire.com on 13/04/2018.
+ */
+public class Orcidv2 implements SolrAuthorityInterface {
+
+
+    public RESTConnector restConnector;
+    private String OAUTHUrl;
+    private String clientId;
+
+    private String clientSecret;
+
+    private String accessToken;
+
+    public void init() throws IOException {
+        if (StringUtils.isNotBlank(accessToken) && StringUtils.isNotBlank(clientSecret)) {
+            String authenticationParameters = "?client_id=" + clientId + "&client_secret=" + clientSecret + "&scope=/read-public&grant_type=client_credentials";
```

Comment:
Where do these credentials (clientId, clientSecret) come from? Are there some config defaults I'm missing? I can't see reference to them in the spring/api/orcid-authority-services.xml spring config or any other config files
(@jonas-atmire , can you shed some light here?)